packages/base/src/keybindings.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "command": "jupytercad:removeObject",
+    "keys": ["Accel X"],
+    "selector": ".jp-jupytercad-panel"
```

Comment:
This results in the key binding to not be triggered if it's the left panel (object list) that has focus.

I wonder if we should add the keybinding to the `#main` application instead. Though I tested locally and that does not seem to work, I suppose it's because the command depends on the `WidgetTracker<JupyterCadWidget>` tracker, and that one only tracks the 3D view if I'm correct.

Let's not consider this a blocker for merging the PR, but let's keep track of this in an issue after merging.

Reply:
I see, i can try adding the valid className for left panel too and see if the union of `jupytercad-panel` & `leftpanel` works.

Reply:
Sounds good!